### PR TITLE
Add ability to pass additional input attributes

### PR DESCRIPTION
**Pull request by juliettepretot**
Would be super helpful for adding attributes such as `maxlength` or `name`.

### CODE REVIEW

File: lib/ReactTags.js [hunk 1]
```diff
@@ -176,6 +176,7 @@ class ReactTags extends React.Component {
           onInput={this.handleInput.bind(this)}
           onKeyDown={this.handleKeyDown.bind(this)}>
           <Input {...this.state}
+            inputAttributes={this.inputAttributes}
```
Comment (i-like-robots) [inline lib/ReactTags.js]: I think this is meant to be `this.props.inputAttributes` =]

**Comment by i-like-robots** [comment]
As discussed in #132 I think this is a good feature, thanks for submitting!

File: README.md [hunk 2]
```diff
@@ -84,6 +84,7 @@ React.render(<App />, document.getElementById('app'))
 - [`handleBlur`](#handleblur-optional)
 - [`allowNew`](#allownew-optional)
 - [`tagComponent`](#tagcomponent-optional)
+- [`inputAttributes`](#inputAttributes-optional)
```
[the remark this answers is not in the document]
Comment (i-like-robots) [reply README.md]: I think I prefer `inputAttributes`

File: README.md [hunk 3]
```diff
@@ -214,6 +215,12 @@ Disables ability to delete the selected tags when backspace is pressed while foc
 
 Provide a custom tag component to render. Default: `null`.
 
+#### inputAttributes (optional)
+
+An object containing additional attributes that will be applied to the underlying `<input />` field.
+
+As an example `inputAttributes={ maxlength: 10 }` would be applied as `<input maxlength="10" ... />`. Note this prop won't overwrite existing attributes, it can only add new ones.
```
[the remark this answers is not in the document]
Comment (i-like-robots) [reply README.md]: I am concerned about overwriting attributes added by the component. If the component is adding them then they should be there for a functional reason.

My optimistic view is that folks would only use this feature if they know what they're doing but my pessimistic view is that the component should - where possible - be defensive.

[the remark this answers is not in the document]
Comment (HugoGiraudel) [reply README.md]: Fair enough. We can keep it like @juliettepretot authored it then. :)

**Comment by HugoGiraudel** [comment]
@i-like-robots Alright. I think we should be good to go on our side. :)

**Comment by i-like-robots** [comment]
Released as v5.6.0

**Comment by HugoGiraudel** [comment]
Thanks!

In v6, the `placeholder` prop should probably no longer be accepted outside of `inputAttributes`.

**Comment by i-like-robots** [comment]
No problem. Yes I'll have to consider how this all ties in together, I'm sure it will be quite a clean result. I've merged the change into the V6 branch to get that underway.

**Pull request merged**